style.css

```diff
@@ -781,3 +781,153 @@ p {
 .input1 > label {
   display: none;
 }
+
+/* Mobile Popup */
+
+.popUp {
+  position: fixed;
+  top: 0;
+  background-color: white;
+  padding: 24px;
+}
```

Comment:
- [ ] Please 🙏🏻 be adhere to the project requirement and fix your `popUp` layout properly because your popUp is going beyond the mobile screen viewport, which makes your design a little unprofessional and also causes a bad user experience because a person can't see the buttons of the popUp menu until he/she needs to scroll but in a worst case scenario lets say if user is not aware about the scrolling the he/she will assume that it is just popUp menu with no option for see project live or get **source** code. Kindly also manage the space properly between the child elements.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-21 17-05-33](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/8599a208-3bbf-4328-9ac4-e3b5b1715a27)

</td>
<td>

![Screenshot from 2023-06-21 17-04-48](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/921fb80a-1a59-4858-a5cb-903e1e4fec03)

</td>
</tr>
</table>